content/concept-transaction-cost.md

```diff
@@ -105,11 +105,11 @@ _Fee levels_ represent the proportional difference between the minimum cost and
 | [EscrowFinish transaction](reference-transaction-format.html#escrowfinish) with 32-byte preimage. | 350 | 256 | 700 | 512 |
 
 
-## Querying the Transaction Cost ##
+## Querying the Transaction Cost
 
 The `rippled` APIs have two ways to query the local load-based transaction cost: the `server_info` command (intended for humans) and the `server_state` command (intended for machines).
 
-If the [FeeEscalation amendment](concept-amendments.html#feeescalation) is enabled, you can use the [`fee` command](reference-rippled.html#fee) to check the open ledger cost.
+If the [FeeEscalation amendment](reference-amendments.html#feeescalation) is enabled, you can use the [`fee` command](reference-rippled.html#fee) to check the open ledger cost.
 
 ### server_info ###
```

Comment:
Remove the trailing #s? Tiny - but wanted to support the formatting cleanup you've done throughout.